src/status_im/contexts/onboarding/welcome/style.cljs

```diff
@@ -8,10 +8,15 @@
    :padding-top (:top insets)})
 
 (defn page-illustration
-  [width]
-  {:width         width
-   :margin-top    12
-   :margin-bottom 4})
+  [{:keys [width height]}]
+  {:width  width
+   ;; Aspect ratio 2:3
+   :height (/ height 1.5)})
```

Comment:
Aspect ratio can be specified as a key in the styles

```suggestion
  {:width  width
   :aspect-ratio 1.5
   :height height})
```

Reply:
thanks, resolved